### PR TITLE
Prefix

### PR DESCRIPTION
**Pull request by catalamarti**

### CODE REVIEW

**Comment by catalamarti** [comment]
hi @edward-burn @ablack3 can we agree on a way so that we create intermediate permanent tables in the analytical packages and how we drop them?

My proposal for the moment is:
1) At the begining of the function we sort a random prefix:
```
prefix <- randomPrefix()
```
2) Every time that we compute an intermediate table, we use this prefix:
```
x |> dplyr::compute(name = uniqueTableName(prefix), temporary = FALSE)
```
3) At the end of the analytical code we drop them:
```
dropTable(cdm = cdm, name = dplyr::starts_with(prefix))
```

Maybe as a suggestion we can make the random prefix to always start with `tmp` so then if we a permanent table in the database is a permanent table that didn't got dropped due to an error and we can drop it manually.

What do you think?

**Comment by edward-burn** [comment]
@catalamarti sounds good to me!

File: R/compute.R [hunk 1]
```diff
@@ -54,14 +54,44 @@ compute.local_cdm <- function(x, ...) {
...
+
+#' Create a random prefix for tables, that contains a random sample of lowercase
+#' letters.
+#'
+#' @param n Number of characters.
+#'
+#' @return A random prefix.
+#' @export
+#'
+randomPrefix <- function(n = 5) {
```
Comment (catalamarti) [inline R/compute.R:79]: remove random one

**Comment by catalamarti** [comment]
Yes it would be nice @ablack3
have you seen the reexport guide? I am happy to have some feedback https://darwin-eu-dev.github.io/omopgenerics/articles/reexport.html

**Comment by ablack3** [comment]
not yet! looks very nice!

**Pull request merged**